cli/src/pcluster/config/cluster_config.py

```diff
@@ -1103,14 +1130,15 @@ def vpc_id(self):
         return AWSApi.instance().ec2.get_subnet_vpc(self.head_node.networking.subnet_id)
 
     @property
-    def ami_id(self):
-        """Get the image id of the cluster."""
+    def headnode_ami_id(self):
```

Comment:
let's rename it to headnode_ami to be aligned with custom_ami and official_ami params.

Reply:
done